src/main.ts

```diff
@@ -23,7 +25,23 @@ export const run = async () => {
 
         let inlineScript: string = core.getInput('inlineScript', { required: true });
         let azcliversion: string = core.getInput('azcliversion', { required: true }).trim().toLowerCase();

```

Comment:
Here required is marked as true, but shouldn't this be optional input?
Also if the  azcliversion not provided then it should take default which should be agentazcliversion

Reply:
I also found this weird, ill see if setting false is works too.

Reply:
have you checked it?

Reply:
yes it works with false too.

Reply:
change it to false as it is optional